doc/api/crypto.md

```diff
@@ -2149,12 +2149,10 @@ added: v10.5.0
 - `salt` {string|Buffer|TypedArray}
 - `keylen` {number}
 - `options` {Object}
-  - `N` {number} CPU/memory cost parameter. Must be a power of two greater
-                 than one. **Default:** `16384`.
+  - `N` {number} CPU/memory cost parameter. Must be a power of two greater than one. **Default:** `16384`.
```

Comment:
Please, wrap long lines at 80 characters or less. Just align with the first list item character, like this:
```md
  - `N` {number} CPU/memory cost parameter. Must be a power of two greater than
    one. **Default:** `16384`.
```